kong/runloop/mesh.lua

```diff
@@ -82,6 +82,12 @@ local function init()
     if not ca_row and err == nil then
       -- No CA cert in DB, we are the first cluster member to start
 
+      if kong.db.strategy == "off" then
+        ngx.log(ngx.WARN, "no cluster_ca in declarative configuration: " ..
```

Comment:
Better use the variadic form of this function for the sake of writing reference code